source/_integrations/ping.markdown

```diff
@@ -89,7 +89,7 @@ device_tracker:
 
 {% configuration %}
 hosts:
-  description: List of device names and their corresponding IP address or hostname.
+  description: List of device names and their corresponding IP address or hostname. (Lower case, numbers and underscore only, see [entity names](/configuration/troubleshooting/#entity-names))
```

Comment:
I'd suggest something like:

```
List of device names and their corresponding IP address or hostname. Device names must conform to the standard requirements of lower case, numbers and underscore only - see [entity names](/configuration/troubleshooting/#entity-names).
```

Reply:
Sure, fine with me.

Reply:
Do I need to edit this somewhere or can is there some "accept" button that would do that automatically?